lib/formatValues.js

```diff
@@ -16,11 +17,12 @@ function formatvalues (decl, stylelint) {
   if (!isString) {
     decl.value = decl.value.trim().replace(/\s+/g, ' ')
   }
+  var quotes = getProperty(stylelint, 'string-quotes')
 
-  if (stylelint && stylelint['string-quotes']) {
-    if (stylelint['string-quotes'] === 'double') {
+  if (quotes) {
```

Comment:
& https://github.com/morishitter/stylefmt/pull/221/files#diff-2ce268534080c2744563c4f986b549d8R10

`getProperty()` returns `null` or `undefined` , these checks may be unnecessary.